openstack/objectstorage/v1/objects/testing/requests_test.go

```diff
@@ -323,3 +326,27 @@ func TestObjectCreateParamsWithSeek(t *testing.T) {
 	_, ok = headers["ETag"]
 	th.AssertEquals(t, true, ok)
 }
+
+func TestCreateTempURL(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+
+	// Handle fetching of secret key inside of CreateTempURL
+	containerTesting.HandleGetContainerSuccessfully(t)
+	accountTesting.HandleGetAccountSuccessfully(t)
+	client := fake.ServiceClient()
+
+	// Get dynamically generated port number so that it can be used to compare final URL
+	port := strings.Split(client.Endpoint, ":")[2]
```

Comment:
If we can set port manually to some consistent number across tests, this could be removed

Reply:
You can find an example here: https://github.com/gophercloud/gophercloud/blob/bd999d0da882fe8c5b0077b7af2dcc019c1ab458/openstack/identity/v3/extensions/oauth1/testing/requests_test.go#L248

Reply:
Great, thank you. Fixed